roaringbitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64NavigableMap.java

```diff
@@ -845,6 +847,22 @@ public void testOr_SameBucket() {
     assertEquals(234, left.select(1));
   }
 
+  @Test
+  public void testOr_Static_SameBucket() {
+    Roaring64NavigableMap left = newDefaultCtor();
+    Roaring64NavigableMap right = newDefaultCtor();
+
+    left.addLong(123);
+    right.addLong(234);
+
+    Roaring64NavigableMap result = Roaring64NavigableMap.or(left, right);
```

Comment:
I believe most new test checks for the output, but do not check the input are not mutated.